bouncer/src/repo/manipulator/modeloptimizer/repo_optimizer_multipart.cpp

```diff
@@ -19,17 +19,27 @@
 * Abstract optimizer class
 */
 
+// Include the BVH library first because the mongo drivers (undef_macros.h) #define conflicting names
+
+#include "bvh/bvh.hpp"
+#include "bvh/sweep_sah_builder.hpp"
+
 #include "repo_optimizer_multipart.h"
 #include "../../core/model/bson/repo_bson_factory.h"
 #include "../../core/model/bson/repo_bson_builder.h"
 
+#include "boost/chrono.hpp"
+#include <algorithm>
+
 using namespace repo::manipulator::modeloptimizer;
 
 auto defaultGraph = repo::core::model::RepoScene::GraphType::DEFAULT;
 
-static const size_t  REPO_MP_MAX_FACE_COUNT = 500000;
+static const size_t REPO_MP_MAX_VERTEX_COUNT = 65536;
 static const size_t REPO_MP_MAX_MESHES_IN_SUPERMESH = 5000;
```

Comment:
As far as I can tell this is no longer in use, are we ok with not imposing this restriction?

Looked through the github issues and [this reminded me](https://github.com/3drepo/3drepobouncer/issues/500)